mllib/src/test/scala/org/apache/spark/mllib/feature/Word2VecSuite.scala

```diff
@@ -37,6 +37,12 @@ class Word2VecSuite extends SparkFunSuite with MLlibTestSparkContext {
     assert(syms.length == 2)
     assert(syms(0)._1 == "b")
     assert(syms(1)._1 == "c")
+
+    // Test that model built using Word2Vec, i.e wordVectors and wordIndec
+    // and a Word2VecMap give the same values.
+    val word2VecMap = model.getVectors
+    val newModel = new Word2VecModel(word2VecMap)
+    assert(newModel.getVectors.mapValues(_.toSeq) == word2VecMap.mapValues(_.toSeq))
```

Comment:
Could you change this to compare (word, vector) pairs, rather than just the vectors?
(Also use triple equals ```===```)